javascript/canvas-zoom.js

```diff
@@ -823,20 +829,23 @@
        * Toggles the visibility of certain canvas buttons and tooltips.
        * @param {boolean} hide - Specifies whether to hide (true) or show (false) the elements.
        */
-      const hideCanvasButtons = (hide = false) => {
-        const selectors = [
-          `${activeElement} button[aria-label='Undo']`.parentElement,
-          `${activeElement} .brush`.parentElement,
-          `${activeElement} .canvas-tooltip-info--extension`
-        ];
-
-        selectors.forEach(selector => {
-          const element = document.querySelector(selector);
-
-          element.classList.toggle("canvas-buttons-hidden", hide);
-          element.classList.toggle("canvas-buttons-visible", !hide);
+      function hideCanvasButtons(hide = false) {
```

Comment:
I couldn't save your solution here either, but I optimized what I had and it seems fine